### PR TITLE
Update UG errors

### PR DESCRIPTION
**Pull request by Jeromesyl**
Applies the changes in my comment on #242

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -218,6 +223,8 @@ Format: `delete INDEX`
     * `INDEX` refers to the index number shown in the displayed applicant list. 
     * `INDEX` **must be a positive integer** e.g. 1, 2, 3, …
     * `INDEX` of 0 or less, or indexes greater than `2147483647` will be treated as invalid commands.
+* If you made a mistake deleting an applicant and would like to rectify it, please use the [**Undo**](#undoing-an-action--undo) command.
```
Comment (Chang-CH) [inline docs/UserGuide.md:226]: I think its quite unnecessary to include this for everything. Undo is for when mistakes occur, this seems more repetitive

Comment (Chang-CH) [reply docs/UserGuide.md:226]: If we are linking commands with hyperlinks I think for standardisation sake we should do for everything as well.

**Review by Chang-CH — CHANGES_REQUESTED** [review]
I dont think user guides talk about undo for every function. If we wanted to at best we move undo to the top of the feature list. Linking commands with hyperlinks is nice but we should do that for all other command references for consistency.

File: docs/UserGuide.md (hunk 1 above)
Comment (Jeromesyl) [reply docs/UserGuide.md:226]: Agreed. Undo/redo isn't really a main feature either.

**Review by Chang-CH — APPROVED** [review]

**Review by yunpeng1234 — APPROVED** [review]
LGTM

**Pull request merged**